docs/oauth.md

```diff
@@ -116,44 +163,46 @@ This set of OAuth Config capabilities is not shared with the regular connector c
 
 | Capability Name  | Description | Default | Recommendation |
 | ----  | ------- | --------- | ----------- |
-| OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN | Whether your OAuth provider supports custom domain. i.e. OAuth endpoint host is not fixed. | false | - |
+| OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN | Enable this if your IDP has different URLs. The OAuth config will specify the relative paths for the URLs, and the end user will have to provide the IDP hostname when creating a new connection. **Note: OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN only works for embedded OAuth config. External OAuth config should specify the instance URL in the config file, if required.** | false | - |
```

Comment:
external oauth config shouldn't set OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN cap, and should specify authUri, tokenUri, userInfoUri fields as absolute paths, which contains the custom domain/instance Url. Customers should create multiple external/custom oauth configs for different custom domains/instance URLs.